$MirOS: ports/devel/subversion/patches/patch-configure_ac,v 1.1 2009/07/29 20:31:28 bsiegert Exp $
--- configure.ac.orig	2009-10-13 18:09:30.000000000 +0200
+++ configure.ac	2009-10-31 13:00:43.000000000 +0100
@@ -23,7 +23,7 @@ else
 fi
 AC_SUBST([canonicalized_srcdir])
 
-SWIG_LDFLAGS="$LDFLAGS"
+SWIG_LDFLAGS=""
 AC_SUBST([SWIG_LDFLAGS])
 
 # Generate config.nice early (before the arguments are munged)
